deps.edn

```diff
@@ -61,19 +61,19 @@
   hiccup/hiccup                             {:mvn/version "1.0.5"}              ; HTML templating
   honeysql/honeysql                         {:mvn/version "1.0.461"             ; Transform Clojure data structures to SQL
                                              :exclusions  [org.clojure/clojurescript]}
-  instaparse/instaparse                     {:mvn/version "1.4.10"}             ; Make your own parser
+  instaparse/instaparse                     {:mvn/version "1.4.12"}             ; Make your own parser
   io.forward/yaml                           {:mvn/version "1.0.11"              ; Clojure wrapper for YAML library SnakeYAML. Don't upgrade yet, new version doesn't support Java 8 (see https://github.com/owainlewis/yaml/issues/37)
                                              :exclusions  [org.clojure/clojure
                                                            org.flatland/ordered
                                                            org.yaml/snakeyaml]}
   io.github.resilience4j/resilience4j-retry {:mvn/version "1.7.1"}              ; Support for retrying operations
   javax.xml.bind/jaxb-api                   {:mvn/version "2.4.0-b180830.0359"} ; add the `javax.xml.bind` classes which we're still using but were removed in Java 11
-  joda-time/joda-time                       {:mvn/version "2.10.13"}
-  kixi/stats                                {:mvn/version "0.4.4"               ; Various statistic measures implemented as transducers
+  joda-time/joda-time                       {:mvn/version "2.10.14"}
+  kixi/stats                                {:mvn/version "0.5.5"               ; Various statistic measures implemented as transducers
```

Comment:
I think this might have broken something last time I tried it. Just a heads up in case tests fail on X-Rays